### PR TITLE
docs(home): change grid with column spans

### PR DESCRIPTION
**Pull request by DuncanFaulkner**
update the image for the grid, also created a stackblitz example for creating the grid, included an image of the source code as well. Source code now shows the grid created using the flex-layout library rather than flexbox and CSS.

I've included the images in a folder within the documentation, I would have preferred to have been able to do something like https://cloud.githubusercontent.com/assets/210413/20034148/ but can't see how to do that yet. If there some documentation on how to do this then I will amend this PR.

### CODE REVIEW

**Comment by DuncanFaulkner** [comment]
@googlebot I consent

**Comment by DuncanFaulkner** [comment]
@CaerusKaru is it possible to push this PR through or is there additional things you would like me to fix?

File: docs/documentation/Home.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 ### Featured Demo
 
-One of the hardest features to implement is a grid layout with specific column spans. Our online demo shows how easy 
+One of the hardest features to implement is a grid layout with specific column spans. Our online demo shows how easy
 this is!
 
 Live Demo:
 
-[![Featured](https://cloud.githubusercontent.com/assets/210413/24045163/216dae9c-0aec-11e7-8b22-616a4328eca3.jpg)](https://tburleson-layouts-demos.firebaseapp.com/#/stackoverflow)
+[![Featured](./images/gridWithColumnSpans.png)](https://stackblitz.com/fork/angular-flex-layout-home-grid-demo)
```
Comment (CaerusKaru) [inline docs/documentation/Home.md:26]: We can't link to demos controlled by people outside of the Angular team, sorry.

File: docs/documentation/Home.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 
 Source Code:
 
-[![Feature Source](https://cloud.githubusercontent.com/assets/210413/24045169/247bad3c-0aec-11e7-827a-6ece0775ff39.jpg)](https://github.com/angular/flex-layout/blob/master/src/demo-app/app/stack-overflow/columnSpan.demo.ts#L28)
+[![Open on stackblitz](./images/gridWithColumnSpansSourceCode.png)](https://stackblitz.com/edit/angular-flex-layout-home-grid-demo?file=src/app/app.component.html)
```
Comment (CaerusKaru) [inline docs/documentation/Home.md:30]: Same here re external demos

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,17258 @@
+{
```
Comment (CaerusKaru) [inline package-lock.json]: Please remove this file. We use Yarn in this project, not NPM.

**Review by CaerusKaru — CHANGES_REQUESTED** [review]
New images look good, some link and file issues but otherwise ready to go.

File: package-lock.json (hunk 3 above)
Comment (DuncanFaulkner) [reply package-lock.json]: Removed

File: docs/documentation/Home.md (hunk 2 above)
Comment (DuncanFaulkner) [reply docs/documentation/Home.md:30]: Removed links to an external site and just left image - this image resides in the documentation/ images folder

**Comment by DuncanFaulkner** [comment]
Changes all complete, not sure why CI is not running again

**Comment by DuncanFaulkner** [comment]
Happy to create new samples so these can go on the Angular's stackblitz setup if required?

**Comment by DuncanFaulkner** [comment]
I think the CI is stopping this progressing

**Pull request closed without merging**